### PR TITLE
CB-106: Temporarily disable Drug Order Tab

### PR DESCRIPTION
**Pull request by andela-aolaniran**
# JIRA TICKET NAME:
[CB-106](https://issues.openmrs.org/browse/CB-106):  Temporarily disable Drug Order Tab
## SUMMARY:
1. Temporarily disabled drug order tab
2. Fixed issue with patient attribute dead/alive radio buttons
3. Removed missing taginputs library.

### CODE REVIEW

File: app/js/components/tabs/tabsComponent.jsx [hunk 1]
```diff
@@ -17,7 +17,8 @@ class TabsComponent extends Component {
                 {active: false, name: 'Patient Attributes', divId: 'patient', component: Components.PatientComponent },
                 {active: false, name: 'Encounter', divId: 'encounter', component: Components.EncounterComponent},
                 {active: false, name: 'Programmme Enrollment', divId: 'programme', component: Components.ProgrammeComponent},
-                {active: false, name: 'Drug Order', divId: 'drug', component: Components.DrugOrderComponent },
+                // Temporarily disable the drug order component
```
Comment (dkayiwa) [inline app/js/components/tabs/tabsComponent.jsx]: Why don't you completely remove this?

Comment (andela-aolaniran) [reply app/js/components/tabs/tabsComponent.jsx]: Yea, I would update the PR

Comment (andela-aolaniran) [reply app/js/components/tabs/tabsComponent.jsx]: @dkayiwa I've updated the PR.

File: app/js/components/tabs/tabcomponents/patientComponent.jsx [hunk 2]
```diff
@@ -404,19 +405,31 @@ class PatientComponent extends Component {
...
+                  /> Dead Only
+                </label>
               </div>
             </div>
           </div>
 
           <div className="form-group">
             <div className="col-sm-offset-2 col-sm-6">
               <button type="submit" onClick={this.searchDemographics} className="btn btn-success">Search</button>
-              <button type="reset" onClick={this.resetSearchByDemographics} className="btn btn-default cancelBtn">Reset</button>
```
Comment (dkayiwa) [inline app/js/components/tabs/tabcomponents/patientComponent.jsx:419]: Is this also related to the disabling of the drug order tab?

File: app/js/components/tabs/tabcomponents/patientComponent.jsx [hunk 3]
```diff
@@ -404,19 +405,31 @@ class PatientComponent extends Component {
...
+                  <input
+                    type="radio"
+                    value="alive"
+                    name="livingStatus"
+                    checked={this.state.livingStatus === 'alive'}
+                    onChange={this.toggleLivingStatus}
+                  /> Alive Only
+                </label>
                 <label>
-                  <input type="radio" value="dead" name="livingStatus" onChange={this.toggleLivingStatus} /> Dead Only
```
Comment (dkayiwa) [inline app/js/components/tabs/tabcomponents/patientComponent.jsx:410]: Is this also related to the disabling of the drug order tab?

File: app/js/components/tabs/tabcomponents/patientComponent.jsx [hunk 4]
```diff
@@ -404,19 +405,31 @@ class PatientComponent extends Component {
             <div className="col-sm-offset-2 col-sm-6">
               <div className="checkbox patient-status">
                 <label>
-                  <input type="radio" value="alive" name="livingStatus" onChange={this.toggleLivingStatus} /> Alive Only
```
Comment (dkayiwa) [inline app/js/components/tabs/tabcomponents/patientComponent.jsx:407]: Is this also related to the disabling of the drug order tab?

File: app/js/components/tabs/tabcomponents/patientComponent.jsx [hunk 5]
```diff
@@ -82,7 +82,7 @@ class PatientComponent extends Component {
     // for living people, diedDuring period -> endDate !== now
     const today = new Date();
     const dayFormat = this.getDateString(today.toISOString());
-    const livingStatus = this.state.livingStatus;
```
Comment (dkayiwa) [inline app/js/components/tabs/tabcomponents/patientComponent.jsx:85]: Is this also related to the disabling of the drug order tab?

File: app/js/components/tabs/tabcomponents/patientComponent.jsx [hunk 6]
```diff
@@ -49,7 +49,7 @@ class PatientComponent extends Component {
 
   searchDemographics(event) {
     event.preventDefault();
-    const { gender, minAge, maxAge, startDate, endDate } = this.state;
+    const { gender, minAge, maxAge, startDate, endDate, livingStatus } = this.state;
```
Comment (dkayiwa) [inline app/js/components/tabs/tabcomponents/patientComponent.jsx:52]: Is this also related to the disabling of the drug order tab?

Comment (andela-aolaniran) [reply app/js/components/tabs/tabcomponents/patientComponent.jsx:52]: @dkayiwa No, I listed it in the PR summary. It was one of the glitch we noticed during the demo.

Comment (dkayiwa) [reply app/js/components/tabs/tabcomponents/patientComponent.jsx:52]: Merging for today. But next time, it would be better to have a separate pull request for removing the drug order tab. 😊
